cmd/info.go

```diff
@@ -117,42 +118,169 @@ func info(ctx *cli.Context) error {
 		if err != nil {
 			logger.Fatalf("write message: %s", err)
 		}
-		data := make([]byte, 4)
-		n := readControl(f, data)
-		if n == 1 && data[0] == byte(syscall.EINVAL&0xff) {
-			logger.Fatalf("info is not supported, please upgrade and mount again")
+		err = f.Sync()
```

Comment:
Why we need this?

Reply:
To ensure the request data being flushed into FUSE.

Reply:
It's not needed before. Have you met any problem if there's no `sync`?

Reply:
I have suspected it result in a problem, but I found the true reason later. I'm going to remove this and run tests.

Reply:
Removed now.